### ISSUE TITLE
Use localized date/timeformat

### ISSUE THREAD

**Issue by ChristophT** [loomio/loomio#4393]
As non-english speaking user it would be nice to be able to change the time and date format to the locale used here, e.g. 24h time format and `dd.MM.yyyy` date format.

I'd expect the applied format to be determined either by the locale of each user or to be configurable for a group. Both variants would be fine, depending if you focus on a consistent behaviour for an user over all groups or consistency for all users of a certain group.

See also this [discussion](https://www.loomio.org/d/BuWjifAN/change-time-and-date-format)

**Comment by juliencarnot** [comment]
I'd use user's current locale as the field already exists.

I guess the date-picker will also use these parameters to display first day of week and weekdays initials accordingly, for now it can be confusing.

**Comment by juliencarnot** [comment]
I have confirmed that the datepicker is really hard to use for French users not used to weeks starting on Sundays and not knowing the first letter of the weekdays in English.

I looked at this, to see if I could make md-datepicker use current locale. It seems to be possible, maybe in ./angular/core/config/md_date_locale.coffee, from what I see in [this article](https://blog.angular.io/taking-advantage-of-the-angular-material-datepicker-237e80fa14b3), but I don't really know where to start.
